torch/fx/experimental/_dynamism.py

```diff
@@ -29,14 +29,18 @@ def module_to_nested_dict(module: torch.nn.Module) -> dict[str, Any]:
     self_dict["_modules"] = {}
 
     for attr_name in dir(module):
-        if not attr_name.startswith("_") and not callable(getattr(module, attr_name)):
-            attr_value = getattr(module, attr_name)
-            if (
-                not isinstance(attr_value, torch.nn.Module)
-                and isinstance(attr_value, (int, float, torch.Tensor))
-                and type(attr_value) is not bool
-            ):
-                self_dict[attr_name] = attr_value
+        try:
+            if not attr_name.startswith("_") and not callable(getattr(module, attr_name)):
+                attr_value = getattr(module, attr_name)
+                if (
+                    not isinstance(attr_value, torch.nn.Module)
+                    and isinstance(attr_value, (int, float, torch.Tensor))
+                    and type(attr_value) is not bool
+                ):
+                    self_dict[attr_name] = attr_value
+        except Exception:
```

Comment:
Yikes... shouldn't we narrow this catch to an `(AttributeError, KeyError)`? If we are going to be catching all exceptions we should at least add logger.debug() so we know something is going wrong and why

Reply:
Oops this wasn't ready for review

Reply:
Also for posterity the case in prod that this fixes is when we have a @property method that throws a NotImplementedError